daprdocs/content/en/getting-started/quickstarts/serviceinvocation-quickstart.md

```diff
@@ -10,12 +10,9 @@ With [Dapr's Service Invocation building block](https://docs.dapr.io/developing-
 
 <img src="/images/serviceinvocation-quickstart/service-invocation-overview.png" width=800 alt="Diagram showing the steps of service invocation" style="padding-bottom:25px;">
 
-Dapr offers the invoke url and proxy methods for service invocation, depending on your scenario. We recommend using proxy, as it's easier for you to:
+Dapr offers several methods for service invocation, which you can choose depending on your individual scenario. For this quickstart, you'll enable the `checkout` service to invoke a method using HTTP proxy in the `order-processor` service.
```

Comment:
Suggest removing "individual". Dapr offers several methods for service invocation, which you can choose depending on your scenario. For this quickstart, you'll enable the `checkout` service to invoke a method using HTTP proxy in the `order-processor` service.